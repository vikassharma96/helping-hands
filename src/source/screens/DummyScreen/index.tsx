import React from 'react';

const DummyScreen: React.FC = () => {
  return (
    <div className="DummyScreen">
      <h1>Helping Hands</h1>
    </div>
  );
};

export default DummyScreen;
