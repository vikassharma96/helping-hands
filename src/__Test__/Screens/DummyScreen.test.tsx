import React from 'react';
import renderer from 'react-test-renderer';
import DummyScreen from '../../source/Screens/DummyScreen';

it('should match snapshot', () => {
  const tree = renderer.create(<DummyScreen />).toJSON();
  expect(tree).toMatchSnapshot()
});
