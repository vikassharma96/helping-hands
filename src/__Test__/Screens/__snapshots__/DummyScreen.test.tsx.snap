// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<div
  className="DummyScreen"
>
  <h1>
    Helping Hands
  </h1>
</div>
`;
